src/collective/cover/tiles/carousel.py

```diff
@@ -66,7 +65,10 @@ def populate_with_object(self, obj):
         :type uuids: List of strings
         """
         super(ListTile, self).populate_with_object(obj)  # check permission
-        uuids = ICoverUIDsProvider(obj).getUIDs()
+        if obj.portal_type == 'Collection':
```

Comment:
All IDGB tiles that accept a Collection to do this "magic", accept a Folder as well. We know brasil.gov.tiles and collective.cover are completely different projects, but we thought that since you don't limit "Folder" in `accepted_ct` in carousel, it would make sense to have this feature for both Collection and Folder.